scripts/fetch_metadata.py

```diff
@@ -0,0 +1,53 @@
+# A script for fetching metadata from video file
+
+import cv2
+import sys
+import os.path
+import json
+import logging
+
+if len(sys.argv) < 2:
```

Comment:
Use argparse it will do this check for you plus type check too and a hint for the field is